src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -824,7 +858,8 @@ private InputsAndSuccessFlag selectInputs(TypedOperation operation) {
         }
       }
 
-      variables.add(totStatements + randomVariable.index);
+      variables.add(
+          totStatements + randomVariable.index + grtFuzzingAndNumStatements.numStatements);
```

Comment:
Is the `GrtFuzzingAndNumStatements` data structure needed?  It seems to me that the number of statements could be computed by subtracting the length of `chosenSeq` before and after the call to `GrtFuzzing.fuzz`.